server/src/internalClusterTest/java/org/elasticsearch/snapshots/ConcurrentSnapshotsIT.java

```diff
@@ -435,7 +435,7 @@ public void testMasterFailOverWithQueuedDeletes() throws Exception {
         final ActionFuture<AcknowledgedResponse> firstDeleteFuture = startDeleteFromNonMasterClient(repoName, firstSnapshot);
         awaitNDeletionsInProgress(1);
 
-        blockDataNode(repoName, dataNode2);
+        blockNodeOnAnyFiles(repoName, dataNode2);
```

Comment:
The fact that we would block on data files here even though snapshot `3`  and snapshot `2` were snapshotting the same data actually already demonstrated the bug in hindsight. Since this test forces an exception on this snapshot below this was hidden from us unfortunately.